tests/listsAndDropDowns.spec.ts

```diff
@@ -4,15 +4,16 @@ test.describe('Lists and Drop Downs', () => {
     //declare variables
     let petTypeInputField
     let petNameInputField
+    let petTypeField
```

Comment:
To comply with TypeScript standards, when you declare the variable without assigning the value, need to declare the type of this variable.
So it will be like 
`let petTypeField: Locator`